tutorials/roofit/rf614_binned_fit_problems.C

```diff
@@ -1,195 +0,0 @@
-      integ->getVal();
-      dataH->set(iBin, nEvents * integ->getVal(), -1);
-   }
-   return dataH;
-}
-
-// Force numeric integration and do this numeric integration with the
-// RooBinIntegrator, which sums the function values at the bin centers.
-void enableBinIntegrator(RooAbsReal &func, int numBins)
-{
-   RooNumIntConfig customConfig(*func.getIntegratorConfig());
-   customConfig.method1D().setLabel("RooBinIntegrator");
-   customConfig.getConfigSection("RooBinIntegrator").setRealValue("numBins", numBins);
-   func.setIntegratorConfig(customConfig);
-   func.forceNumInt(true);
-}
-
-// Reset the integrator config to disable the RooBinIntegrator.
-void disableBinIntegrator(RooAbsReal &func)
-{
-   func.setIntegratorConfig();
-   func.forceNumInt(false);
-}
-
-void rf614_binned_fit_problems()
-{
-   using namespace RooFit;
-
-   // Silence info output for this tutorial
-   RooMsgService::instance().getStream(1).removeTopic(Minimization);
-   RooMsgService::instance().getStream(1).removeTopic(Fitting);
-   RooMsgService::instance().getStream(1).removeTopic(Generation);
-
-   // Exponential example
-   // -------------------
-
-   // Set up the observable
-   RooRealVar x{"x", "x", 0.1, 5.1};
-   x.setBins(10); // fewer bins so we have larger binning effects for this demo
-
-   // Let's first look at the example of an exponential function
-   RooRealVar c{"c", "c", -1.8, -5, 5};
-   RooExponential expo{"expo", "expo", x, c};
-
-   // Generate an Asimov dataset such that the only difference between the fit
-   // result and the true parameters comes from binning effects.
-   std::unique_ptr<RooAbsData> expoData{generateBinnedAsimov(expo, x, 10000)};
-
-   // If you do the fit the usual was in RooFit, you will get a bias in the
-   // result. This is because the continuous, normalized pdf is evaluated only
-   // at the bin centers.
-   std::unique_ptr<RooFitResult> fit1{expo.fitTo(*expoData, Save(), PrintLevel(-1), SumW2Error(false))};
-   fit1->Print();
-
-   // In the case of an exponential function, the bias that you get by
-   // evaluating the pdf only at the bin centers is a constant scale factor in
-   // each bin. Here, we can do a trick to get rid of the bias: we also
-   // evaluate the normalization integral for the pdf the same way, i.e.,
-   // summing the values of the unnormalized pdf at the bin centers. Like this
-   // the bias cancels out. You can achieve this by customizing the way how the
-   // pdf is integrated (see also the rf901_numintconfig tutorial).
-   enableBinIntegrator(expo, x.numBins());
-   std::unique_ptr<RooFitResult> fit2{expo.fitTo(*expoData, Save(), PrintLevel(-1), SumW2Error(false))};
-   fit2->Print();
-   disableBinIntegrator(expo);
-
-   // Power law example
-   // -----------------
-
-   // Let's not look at another example: a power law \f[x^a\f].
-   RooRealVar a{"a", "a", -0.3, -5.0, 5.0};
-   RooPower powerlaw{"powerlaw", "powerlaw", x, RooConst(1.0), a};
```

Comment:
Unfortunately, this tutorial uses the `RooPower` that this PR suggests to remove.

One could have also adapted the tutorial to show a different example, but this divergence of tutorials between branches would probably be more confusing.

People usually only look at the latest tutorials anyway, as a google search for "RooFit tutorials" sends you to the `master` version:
https://root.cern.ch/doc/master/group__tutorial__roofit.html